daemon/run_openrazer_daemon.py

```diff
@@ -138,6 +142,14 @@ def run():
         stop_daemon(args)
         time.sleep(3)
 
+    # daemonize logs exceptions to its logger (which defaults to the syslog)
+    # and does not make them appear on stdout/stderr. If we're in foreground
+    # mode, override that logger with our own.
+    if not args.foreground:
```

Comment:
Shouldn't it be `if args.foreground:` as we **want** our own logger in foreground mode? If yes, don't bother making a PR, I can fix it myself...

Reply:
oops. some testing code that stuck in via the last commit --amend, sorry about that. Correct version is as you said, thanks.